index.html

```diff
@@ -5,7 +5,7 @@
 </head>
 <script>
 // JS for the tabs
-function openCity(evt, cityName) {
+function openCity(evt, tab) {
```

Comment:
please rename this away from `...City`

Reply:
Sorry, missed this. Will fix in next pull request im'h,

Reply:
> Sorry, missed this. Will fix in next pull request im'h,

Please do it in THIS pull request, not the next one.

Reply:
Done.

Reply:
Please rename `openCity` to something that does not mention cities.

Reply:
Oh no! Am I editing the wrong branch? I fixed this issue, but not here??